AirCasting/SDSync/SDSyncFileValidationService.swift

```diff
@@ -8,29 +8,86 @@ enum SDCardValidationError: Error {
 }
 
 protocol SDSyncFileValidator {
-    func validate(files: [(URL, SDCardSessionType, expectedMeasurementsCount: Int)], completion: (Result<Void, SDCardValidationError>) -> Void)
+    func validate(files: [SDCardCSVFile], completion: (Result<Void, SDCardValidationError>) -> Void)
 }
 
 struct SDSyncFileValidationService: SDSyncFileValidator {
-    func validate(files: [(URL, SDCardSessionType, expectedMeasurementsCount: Int)], completion: (Result<Void, SDCardValidationError>) -> Void) {
-        files.forEach { file in
+    private var EXPECTED_FIELDS_COUNT = 13
+    private var ACCEPTANCE_THRESHOLD = 0.8
```

Comment:
we don't use all-caps snake cased constatnts in Swift, just a normal `camelCase`